samples/javascript_nodejs/10.prompt-validations/bot.js

```diff
@@ -64,7 +64,7 @@ class MainDialog {
             // If no response has been sent, start the onboarding dialog.
             if (!turnContext.responded) {
                 await dc.beginDialog(ONBOARD_USER);
-            } 
+            }
         } else if (
             turnContext.activity.type === ActivityTypes.ConversationUpdate &&
```

Comment:
This needs to be cleaned up. Welcoming users section is not correct. Can you bring this section over from the basic-bot sample?

Reply:
Not sure what you want me to do to clean it up.  

If we bring the other class in, that's fine, but would effectively replace this sample which demonstrates multiple different type of custom prompt validators, whereas that one only has string validators.

But I'd like to know what you don't like about how I did this too!

Reply:
Updated!